### PR TITLE
Resetting the translation state of UIPanGestureRecoginzer to 0,0

### PR DESCRIPTION
**Pull request by pittmab**
Fix: 984

### CODE REVIEW

**Comment by rajsesh-msft** [comment]
@jaredhms this one is yours as well.

**Comment by jaredhms** [comment]
@davelamb, please take a look.

File: Frameworks/UIKit/UIPanGestureRecognizer.mm [hunk 1]
```diff
@@ -478,6 +478,7 @@ - (_UIPanGestureStage)_stage {
 */
 - (void)reset {
     _priv->touches.clear();
+	_priv->currentTranslation.x = _priv->currentTranslation.y = 0.f;
```
Comment (jaredhms) [inline Frameworks/UIKit/UIPanGestureRecognizer.mm]: >	 [](start = 0, length = 1)

nit: please switch the tab to 4 spaces

Comment (jaredhms) [inline Frameworks/UIKit/UIPanGestureRecognizer.mm]: >_priv->currentTranslation.x = _priv->currentTranslation.y = 0.f; [](start = 1, length = 64)

nit: I would prefer separate lines for the variable assignments

Comment (pittmab) [inline Frameworks/UIKit/UIPanGestureRecognizer.mm]: @jaredhms The original code that sets the currentTranslation to 0 does a simultaneous assignment.  Should we change both at the same time to be consistent?

Comment (jaredhms) [inline Frameworks/UIKit/UIPanGestureRecognizer.mm]: if you don't mind, that would be great.  

---
In reply to: [78601747](https://github.com/Microsoft/WinObjC/pull/985#discussion_r78601747) [](ancestors = 78601747)

Comment (pittmab) [inline Frameworks/UIKit/UIPanGestureRecognizer.mm]: @jaredhms Changes have been made.

**Comment by jaredhms** [comment]
:shipit:

**Comment by jaredhms** [comment]
@davelamb is added to the review. #Closed

**Pull request merged**